src/Files.App/Data/Commands/RichGlyph.cs

```diff
@@ -54,22 +55,22 @@ public void Deconstruct(out string baseGlyph, out string fontFamily, out string
 			return fontIcon;
 		}
 
-		public OpacityIcon? ToOpacityIcon()
+		public ThemedIcon? ToThemedIcon()
 		{
-			if (string.IsNullOrEmpty(OpacityStyle))
+			if (string.IsNullOrEmpty(ThemedIconStyle))
 				return null;
 
 			return new()
 			{
-				Style = (Style)Application.Current.Resources[OpacityStyle]
+				Style = (Style)Application.Current.Resources[ThemedIconStyle]
 			};
 		}
 
-		public Style? ToOpacityStyle()
+		public Style? ToThemedIconStyle()
 		{
-			if (string.IsNullOrEmpty(OpacityStyle))
+			if (string.IsNullOrEmpty(ThemedIconStyle))
 				return null;
-			return (Style)Application.Current.Resources[OpacityStyle];
+			return (Style)Application.Current.Resources[ThemedIconStyle];
```

Comment:
This causes an exception to occur

Reply:
I traced the issue down to the icons in `src/Files.App/Actions/Display/LayoutAction.cs`.